courses/cunix/ex04/src/my_atoi.c

```diff
@@ -0,0 +1,32 @@
+int my_atoi(const char *nptr)
+{
+
+    const char *s = nptr;
+    int n = 0;
+    int sign = 0;
+
+    if (*s == '-')
+    {
+        sign = -1;
+    }
+
+    else
+    {
+        sign = 1;
+    }
```

Comment:
```suggestion
    if (*s == '-') {
        sign = -1;
    } else {
        sign = 1;
    }
```
code style could be much better. It's is also very important because when you're working on a project with a team the code style is same for everyone. It makes code readable and organized.